translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -452,3 +456,28 @@ func getTagsFromTraceState(traceState pdata.TraceState) ([]model.KeyValue, bool)
 	}
 	return keyValues, exists
 }
+
+func getTagsFromInstrumentationLibrary(il pdata.InstrumentationLibrary) ([]model.KeyValue, bool) {
+	keyValues := make([]model.KeyValue, 0)
+	if il.IsNil() {
+		return keyValues, false
```

Comment:
`return nil`

Reply:
I followed the example of existing `getTagsFromTraceState` function. And in fact all other functions in this file return some object and a boolean. I can do as you say, no problem, but isn't consistency important? :)

Reply:
I did't look at other functions :) this one seems to be cleaner if simplified, but the consistency is a valid point. feel free to keep it as is